docs/core/compatibility/index.md

```diff
@@ -0,0 +1,316 @@
+
+- **❌ Firing an existing event when it was never fired before**
+
+## Behavioral changes
+
+### Assemblies
+
+- **✔️ Making an assembly portable when the same platforms are still supported**
+
+- **❌ Changing the name of an assembly
+**
+- **❌ Changing the public key of an assembly**
+
+### Properties, fields, parameters, and return values
+
+- **✔️ Changing the value of a property, field, return value, or [out](../../csharp/language-reference/keywords/out-parameter-modifier.md) parameter to a more derived type**
+
+  For example, a method that returns an type of <xref:System.Object> can return a <xref:System.String> instance. (However, the method signature cannot change.)
+
+- **✔️ Increasing the range of accepted values for a property or parameter if the member is not [virtual](../../csharp/language-reference/keywords/virtual.md)**
+
+  Note that while the range of values that can be passed to the method or are returned by the member can expand, the parameter or member type cannot. For example, while the values passed to a method can expand from 0-124 to 0-255, the parameter type cannot change from <xref:System.Byte> to <xref:System.Int32>.
+
+- **❌ Increasing the range of accepted values for a property or parameter if the member is [virtual](../../csharp/language-reference/keywords/virtual.md)**
+
+   This change breaks existing overridden members, which will not function correctly for the extended range of values.
+
+- **❌ Decreasing the range of accepted values for a property or parameter**
+
+- **❌ Increasing the range of returned values for a property, field, return value, or [out](../../csharp/language-reference/keywords/out-parameter-modifier.md) parameter**
+
+- **❌ Changing the returned values for a property, field, method return value, or [out](../../csharp/language-reference/keywords/out-parameter-modifier.md) parameter**
+
+- **❌ Changing the default value of a property, field, or parameter**
+
+- **❌ Changing the precision of a numeric return value**
+
+- **❓ A change in the parsing of input and throwing new exceptions (even if parsing behavior is not specified in the documentation**
+
+### Exceptions
+
+- **✔️ Throwing a more derived exception than an existing exception**
+
+  Because the new exception is a subclass of an existing exception, previous exception handling code continues to handle the exception. For example, in .NET Framework 4, culture creation and retrieval methods began to throw an <xref:System.Globalization.CultureNotFoundException> instead of an <xref:System.ArgumentException> if the culture could not be found. Because  <xref:System.Globalization.CultureNotFoundException> derives from <xref:System.ArgumentException>, this is an acceptable change.
```

Comment:
```suggestion
  Because the new exception is a subclass of an existing exception, previous exception handling code continues to handle the exception. For example, in .NET Framework 4, culture creation and retrieval methods began to throw an <xref:System.Globalization.CultureNotFoundException> instead of an <xref:System.ArgumentException> if the culture could not be found. Because <xref:System.Globalization.CultureNotFoundException> derives from <xref:System.ArgumentException>, this is an acceptable change.
```